schema/alloff_test.go

```diff
@@ -35,7 +35,7 @@ func TestAllOfValidator(t *testing.T) {
 			Name:      `{Bool, String}.Validate("")`,
 			Validator: schema.AllOf{&schema.Bool{}, &schema.String{}},
 			Input:     "",
-			Error:     "not a Boolean",
+			Error:     "not a boolean",
```

Comment:
Boolean is a last name and does always take a cap.

Reply:
Right, from George Boole it seams. Given the Oxford dictionary also agrees with you I suppose I will change it back 👍